resharper/resharper-unity/src/CSharp/Daemon/Stages/PerformanceCriticalCodeAnalysis/PerformanceCriticalCodeAnalysisStage.cs

```diff
@@ -359,6 +366,23 @@ public override void VisitInvocationExpression(IInvocationExpression invocationE
                 }
             }
 
+            private void RegisterImplicitInvocation(IInvocationExpression invocationExpression, HotMethodAnalyzerContext context)
+            {
+                var name = invocationExpression.Reference?.Resolve().DeclaredElement?.ShortName;
+                if (name == null)
+                    return;
+
+                if (name.Equals("Invoke") || name.Equals("InvokeRepeating"))
```

Comment:
No, because this Invoke is already inside performance critical context, so this Invoke will be called each frame, so method which invoked via Invoke should be performance critical context too.

Reply:
OK. Two ways of looking at it - if someone has `Invoke` so that it's called every frame, then yes, we need to mark the invoked call as performance critical, because they're doing something crazy. But if they're calling `Invoke` conditionally, so it's not every frame, then I don't think it's a performance critical context. I was assuming they weren't crazy :)

So I'm happy to go with the worst case scenario here. All good as it is.